### ISSUE TITLE
Voiceover is not announcing as selected/ unselected for Tags and Icons: A11y_.NET 9 template_Project_Screen reader

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/maui#27460]
### GitHub Tags:
#A11yTCS; #A11yMAS; #WCAG1.3.1; #A11ySev2; #iOS; #SH_.NET9template_iOS_Jan2025; #Voiceover; #.NET Core; #Device; #screen reader; #info and Relationship; #AINotInscope; #Element:Buttons; #iphone;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
iOS Version: 18.2
Screen reader: Voiceover

### Repro Steps:

1. Turn on Voiceover and Install and open ".NET 9 template" app.
2. "Home Screen" will open. Swipe till Hamburger menu and double tap on it, Projects options will screen will open.
3. Swipe to any project and double tap to activate it.
4. Project screen will open. Swipe till Icon section and Tags section.
5. Observe that Voiceover is not announcing as selected/ unselected for Tags and Icons

### Actual Result:
Voiceover is not announcing as selected/ unselected when we select any of the Tags and Icons.

### Expected Result:
Voiceover should be announcing as selected/ unselected when we select any of the Tags and Icons.

### User Impact:
Users who rely on screen reader will get confused that those buttons are selected or not selected and will not get appropriate information.

### Attachment:

https://github.com/user-attachments/assets/77f463e2-a1a7-421c-8ae8-63f52ef4ff87

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:07-28-25;

Closing this bug as logged new bug to track this issues.
Bug for reference: https://github.com/dotnet/maui/issues/30749
